HousingSearchListener/V1/UseCase/Interfaces/IAddPersonToTenureUseCase.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace HousingSearchListener.V1.UseCase.Interfaces
+{
+    public interface IAddPersonToTenureUseCase : IMessageProcessing
```

Comment:
is this missing the method from `AddPersonToTenureUseCase` ?

Reply:
No. All the use cases derive implement the same `IMessageProcessing` interface which allows the function handler to create the appropriate concrete use case according to the incoming message type and then call the common method to perform the processing.